services/Cargo.toml

```diff
@@ -37,7 +37,7 @@ flexi_logger = { version = "0.23", features = ["trc"] }
 float-cmp = "0.9"
 futures = "0.3"
 futures-util = "0.3"
-gdal = "0.12"
+gdal =  { git = "https://github.com/georust/gdal.git", branch = "mdarray-future" } # "0.13"
```

Comment:
add todo?

Reply:
idk, we actually should not merge before that :disappointed:

Reply:
[2f72fa4](https://github.com/geo-engine/geoengine/pull/605/commits/2f72fa434ebd52719938ee623771bf4055ec164d)